src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs

```diff
@@ -401,6 +404,17 @@ private ValueTask<HttpConnection> GetOrReserveHttp11ConnectionAsync(Cancellation
                         return (null, true, failureResponse);
                     }
 
+                    if (_kind == HttpConnectionKind.Http)
+                    {
+                        http2Connection = new Http2Connection(this, stream);
+                        await http2Connection.SetupAsync().ConfigureAwait(false);
+
+                        Debug.Assert(_http2Connection == null);
+                        _http2Connection = http2Connection;
+
```

Comment:
The TLS version of this code below has a trace statement.  This block should have one too:

```c#
                        if (NetEventSource.IsEnabled)
                        {
                            Trace("New HTTP2 connection established.");
                        }
```

Reply:
And perhaps the trace text should say "New unencrypted HTTP2 connection established".

Reply:
done.